File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/wardsupply/shared/ward-requisition.model.ts

import { WardRequisitionItemsModel } from './ward-requisition-items.model'
         
export class WardRequisitionModel {
    public RequisitionId: number = 0;
    public WardId: number = 0;
    public StoreId: number = 0;
    public Status: string = '';
    public ReferenceId: string = '';
    public CreatedBy: number = 0;
    public CreatedOn: string = '';
    public WardRequisitionItemsList: Array<WardRequisitionItemsModel> = new Array<WardRequisitionItemsModel>();
    public ItemId: number = 0;
    public Quantity: number = 0;
    public SubTotal: number = 0;
}

File: Code/Components/DanpheEMR.ServerModel/WardSupplyModels/WARDRequisitionModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class WARDRequisitionModel
    {
        [Key]
        public int RequisitionId { get; set; }
        public int WardId { get; set; }
        public int StoreId { get; set; }
        public string Status { get; set; }
        public string ReferenceId { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? CreatedOn { get; set; }
        public virtual List<WARDRequisitionItemsModel> WardRequisitionItemsList { get; set; }
    }
}
